andy/src/main/java/nl/tudelft/cse1110/andy/execution/step/GetRunConfigurationStep.java

```diff
@@ -25,16 +25,43 @@ public void execute(Context ctx, ResultBuilder result) {
             Class<?> runConfigurationClass = Class.forName(getConfigurationClass(ctx.getNewClassNames()), false, currentClassLoader);
             RunConfiguration runConfiguration = (RunConfiguration) runConfigurationClass.getDeclaredConstructor().newInstance();
 
+            if(!isValidRunConfiguration(runConfiguration)) {
+                // load default configuration
+                ctx.setRunConfiguration(new DefaultRunConfiguration(allClassesButTestingAndConfigOnes(ctx.getNewClassNames())));
+                throw new RuntimeException("The run configuration has invalid weights. Please check the file.");
```

Comment:
Let's throw a new exception, maybe `AndyConfigurationException`. Andy should stop in case the configuration is invalid, so in this sense, there's no need to set a default configuration! 

Somewhere up we should catch this exception and generate an error output to the user. The message should clearly indicate this isn't the student's problem but the teacher's problem.